package.json

```diff
@@ -2,38 +2,51 @@
   "name": "molotov",
   "description": "Molotov: (Mix/Plug)-in loader for explosive composeability",
   "main": "molotov.js",
+  "version": "0.0.0-development",
   "directories": {
     "example": "examples"
   },
   "dependencies": {
-    "callsite": "^1.0.0",
-    "config": "^1.25.1",
+    "ajv": "^6.2.0",
     "fs-extra": "^4.0.1",
     "lodash": "^4.17.4",
-    "require-directory": "^2.1.1",
-    "try-require": "^1.2.1"
+    "require-directory": "^2.1.1"
```

Comment:
Are you still using require-directory anywhere? It's not a big problem as long as it's require time and not user time.

Reply:
nope, burn it to the ground 👍